readme.MD

```diff
@@ -1,16 +1,47 @@
 # Dialogs Framework
 The Dialogs Framework is a Python library for writing resumable functions with persistent execution state.
 
-# Examples
+## Resumable what now?
+This framework is intended for long-lived backend flows, such as chatbot conversations. It allows one to write code as a chronological flow, and provides the magic to make execution resume where it left off.
```

Comment:
Give an explanation for what flow is. I would also explain how it is being used in K.

Reply:
I added a paragraph but I couldn't write something clear enough about the K usage, I think it still sounds mysterious to non-users